### ISSUE TITLE
关于训练数据集与验证集的重新划分

### ISSUE THREAD

**Issue by apuomline** [HighwayWu/DeepFakeDefenders#6]
作者您好，我在比赛时也发现了训练集与验证集之间的数据分布很相似。但是我没有做最终的处理。我很想知道这个重新划分代码您能否开源？

**Comment by HighwayWu** [comment]
Thank you for your question and attention. Please try to first perform unsupervised clustering of the data based on its signal characteristics, and then re-partition it using the Leave-One-Out method to force the training set and validation set to contain different types of fake/real data. The relevant code will be open sourced later.
